local-tests/test_multiple_restarts.py

```diff
@@ -11,7 +11,7 @@
 # Path to the aleph-node binary (important DON'T use short-session feature):
 binary = abspath(os.getenv('ALEPH_NODE_BINARY', join(workdir, 'aleph-node')))
 
-phrases = [f'//{i}' for i in range(6)]
+phrases = [f'//{i}' for i in range(5)]
```

Comment:
Huh, why 6→5?

Reply:
It goes with my hypothesis, that 6 nodes is a little bit much for our pipeline. I see the point of 1 non validator which is killed, but I don't think that 1 non validator that is not killed is needed. I believe in the past we had some pipelines failing because they were running too many nodes.

Reply:
Perhaps this should end up in a comment here then, sounds like relevant information and we don't really have it anywhere outside our brains. <_<"

Reply:
This seems like a strange thing for adding here :// there are other tests where this also might be a case (like test_catch_up). Comment in a specific place of code makes sense in context of this change, but seems to me like we should either fully diagnose it and document in a more global place... or maybe this is not the case and we have some strange bug that makes this test more flaky on github. Also maybe this comment should be added in all places where we define the number of validators for a test.

Reply:
Hm, very good point. I'm not sure where the info should be either... Perhaps some constant (`MAX_PIPELINE_NODES`?) in a common file for the tests? Do we even have such a thing?

Reply:
Actually I don't think we will decrease the number of nodes. Test seem flaky anyway :( This would be more fixed after we implement wait_for_finalization in tests